src/lib/settings.js

```diff
@@ -59,6 +69,7 @@ export const DOM_ATTRIBUTES = {
     SETTING_HOVERING:       'data-go-hovering',
     PLAYER_PLACEMENT:       'data-go-place',
     OPPONENT_PLACEMENT:     'data-go-opponent',
+    OPPONENT_NAME:          'data-go-name',
```

Comment:
I'd suggest naming the attribute as `data-go-opponent-name` if it's opponent name, or variable to `PLAYER_NAME` if it's player name :)